central/sensor/augmentation/broker.go

```diff
@@ -0,0 +1,88 @@
+package augmentation
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/sensor/service/connection"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/uuid"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+// The Broker coordinates and matches deployment enhancement requests to responses
+type Broker struct {
+	requests map[string]chan<- *central.DeploymentEnhancementResponse
+	lock     sync.Mutex
+	once     sync.Once
+}
+
+// NewBroker returns a new broker
+func NewBroker() *Broker {
+	return &Broker{
+		requests: make(map[string]chan<- *central.DeploymentEnhancementResponse),
+	}
+}
+
+// NotifyDeploymentReceived matches the ID of Sensors response to the request and notifies the waiting goroutine
```

Comment:
```suggestion
// NotifyDeploymentReceived matches the ID of Sensors response to the request and notifies the waiting goroutine  that the dinner time has come
```
I would like to have some more concrete details here :)